pages/_en-US/nds-bootstrap/faq.md

```diff
@@ -40,15 +40,12 @@ You can get nightly builds for nds-bootstrap [here](https://github.com/TWLBot/Bu
 The way E cheat types are implemented in nds-bootstrap is broken, meaning they'd only work half of the time.
 Your cheat probably uses that type. It is not a fault of the cheat database, but rather a fault of nds-bootstrap. Please do not request these cheats to get deleted from the DB.
 
-#### Is there anything else I should know about cheats?
-You can set heap shrink to `Hi` in settings for slightly better cheat support.
-
 For more info on cheats, look [here](https://wiki.ds-homebrew.com/ds-index/retail-roms#action-replay-cheats).
 
 #### nds-bootstrap troubleshooting
 If you get issues with using the latest version of nds-bootstrap, here's how you could troubleshoot it.
 
-1. Run everything in 67 MHz, DS Mode, with 32 kHz of audio, etc. Basically, keep everything to DS settings. Don't try to improve it with DSi stuff
+1. Run the game with different settings, including changing the ARM9 CPU Speed, DS / DSi Mode, sound quality, Card read DMA, etc
```

Comment:
"Card read DMA" should probably be either "Card Read DMA" or "card read DMA", the former is probably more fitting with the other things here.

Other than that (and the same thing on testing.md#17) this looks good to me.